### PR TITLE
[TACHYON-1650] Add the Chinese version of documentation page 'Running-Tachyon-Fault-Tolerant-on-EC2'

### PR DESCRIPTION
**Pull request by saltylin**
https://tachyon.atlassian.net/browse/TACHYON-1650

Add the Chinese version of documentation page 'Running-Tachyon-Fault-Tolerant-on-EC2'

### CODE REVIEW

File: docs/cn/Running-Tachyon-Fault-Tolerant-on-EC2.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+layout: global
+title: 在EC2上运行带容错机制的Tachyon
+nickname: 运行带容错机制的Tachyon
+group: User Guide
+priority: 4
+---
+
+使用Tachyon自带的[Vagrant脚本](https://github.com/amplab/tachyon/tree/master/deploy/vagrant)可以将带容错机制的Tachyon部署在Amazon EC2上。该脚本允许你创建，配置以及销毁集群，该集群自动配置了HDFS相关项。
+
+# 前期准备
```
Comment (yupeng9) [inline docs/cn/Running-Tachyon-Fault-Tolerant-on-EC2.md]: It seems this section is used in multiple docs. What do you think of extracting it into a snippet under `_includes`?

**Comment by yupeng9** [comment]
Thanks for the translation. I left one question.

**Comment by saltylin** [comment]
@yupeng9 I also see this. In fact, many pages share some sentences, and it just happens that this sentence seems to be longer. Now our design is sharing same code fragments and table elements in the same pages of different languages. Thus for different pages of one particular language they are independent. If they also share snippets under `_includes/`, All pages may get tightly coupled. What's more, maybe this sentence is little longer so it seems necessary to extract it out. However, what about a shorter sentence (since many short sentences are shared among different pages)? I think it's better to follow a simple rule, rather than depending on the sentence' length.

**Comment by yupeng9** [comment]
Fair points. Thanks.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**